conference.js

```diff
@@ -827,25 +841,28 @@ export default {
 
             this._displayErrorsForCreateInitialLocalTracks(errors);
 
-            return this._setLocalAudioVideoStreams(tracks);
+            let localTracks = handleStartAudioMuted(initialOptions, tracks);
+
+            // in case where gum is slow and resolves after the startAudio/VideoMuted coming from jicofo, we can be
```

Comment:
isn't it the body of `handleStartAudioMuted`?

Reply:
ah no it's not

Reply:
Yeah, first commit is that startWithAudioMuted url param was ignored. The second one is that jicofo's startAudioMuted is ignored.